sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizeEntityAsyncClient.java

```diff
@@ -97,7 +98,7 @@
                 documentEntities.getStatistics() == null ? null
                     : toTextDocumentStatistics(documentEntities.getStatistics()),
                 null, documentEntities.getEntities().stream().map(entity ->
-                new NamedEntity(entity.getText(), entity.getType(), entity.getSubtype(), entity.getOffset(),
+                new CategorizedEntity(entity.getText(), entity.getType(), entity.getSubtype(), entity.getOffset(),
```

Comment:
Should these be `entity.getCategory()` and `entity.getSubCategory()`?

Reply:
So this the service `entity` (generated code) model class that we are converting to `CategorizedEntity`, hence the `getType()`

Reply:
Got it, figured it was fine, but just wanted to make sure